### DISCUSSION TITLE
Gradle Configuration Not Working

### DISCUSSION THREAD

**Discussion by DeletedAcc0unt** [x86xFX/Android-ViewBinding#4] [Q&A] [answered]
Hello,
```
android {
    buildFeatures {
      viewBinding true
   }
}
```
This `build.gradle`configuration not work for me. After this setup bindingFiles not generating but if i use databinding true, its generate binding files. Why?

**Comment by x86xFX** [comment, marked as answer]
No its works.
- Change only **module** `build.gradle`
- Do not use `dataBinding true` for **ViewBinding**
- After that sync your gradle files.
- Try both ways

![image](https://user-images.githubusercontent.com/101990722/194540324-f4474aaf-c586-47ad-a04a-cf5aca8b41c0.png)

 If your layout name is `my_custom_layout.xml` then it rename to **Pascal Case** binding file.

```
my_custom_layout.xml  --> MyCustomLayoutBinding.java
```

Sometimes suggestion not working 100%. Then try typing `bind`:

![image](https://user-images.githubusercontent.com/101990722/194541206-08e91f04-f837-4c4a-b818-39968d4d5feb.png)

`private lateinit var binding: MyCustomLayoutBinding`

**Reply by DeletedAcc0unt** [reply]
It works!
Typing `bind` for suggestions is working for me.
Thanks
